conda/recipes/librapidsmpf/install_librapidsmpf.sh

```diff
@@ -2,4 +2,4 @@
 # SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES.
 # SPDX-License-Identifier: Apache-2.0
 
-cmake --install cpp/build
+cmake --install cpp/build --component rapidsmpf
```

Comment:
I think this may be more of a hot-fix.

The [`cpp/tests/CMakeLists.txt`](https://github.com/rapidsai/rapidsmpf/blob/branch-25.06/cpp/tests/CMakeLists.txt) installs it's components also without `--component=testing` right now.  Not sure exactly why, maybe as it doesn't use `rapids_test_add`/`rapids_test_install_relocatable` (but it's a bit more complicated).

On the plus side, excluding the tests almost halves the size of the `librapidsmpf` package :).

Reply:
I agree about tests, but I think it's useful to have benchmarks in the package. How about adding benchmarks to a separate component (currently they're part of `testing`) and including those in the package as well?

Reply:
OK, that also made me realize that all that was missing from testing was `EXCLUDE_FROM_ALL`, so now benchmarking is its own component, but explicitly included in the package.